diff --git a/conanfile.py b/conanfile.py
index 22af95074..b5f8e8b91 100644
--- a/conanfile.py
+++ b/conanfile.py
@@ -101,7 +101,7 @@ class UserverConan(ConanFile):
             self.requires('jemalloc/5.3.0')
         if self.options.with_grpc:
             self.requires(
-                'grpc/1.48.4', transitive_headers=True, transitive_libs=True,
+                'grpc/1.54.2', transitive_headers=True, transitive_libs=True,
             )
             self.requires(
                 'googleapis/cci.20230501',
